eWAY.Rapid/Properties/AssemblyInfo.cs

```diff
@@ -32,8 +32,8 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.6.3.0")]
-[assembly: AssemblyFileVersion("1.6.3.0")]
+//[assembly: AssemblyVersion("1.6.4.0")]
```

Comment:
AssemblyVersion is incremented but also commented out?

Reply:
@J-Lam-Vii ,
comment them out due to duplicate compilation issue.
And the version info is put to eWAY.Rapid.csproj project file